gradle/sonar.gradle

```diff
@@ -0,0 +1,19 @@
+apply plugin: 'org.sonarqube'
+
+def analysisMode = 'publish'
+//TODO(kzaikin): use preview mode in pull requests
+//if (new Boolean(System.env.TRAVIS_PULL_REQUEST)) {
+//    analysisMode = 'preview'
+//}
+
+sonarqube {
+    properties {
+        property 'sonar.analysis.mode', analysisMode
+        property 'sonar.host.url', System.env.SONAR_HOST_URL
+        property 'sonar.login', '11cab05437bb5e07221bd50afc1e694ea8f64b49'
```

Comment:
as soon as I have specified this temporary sonar token I was able to run sonar analysis of the project:
https://sonarcloud.io/dashboard?id=com.github.spotbugs.spotbugs

Looks like we have some problems with travis-ci secret values. What about specifying SONAR_LOGIN environment variable in travis-ci's project properties?